src/ProjectTemplates/Web.ProjectTemplates/GrpcApplication-CSharp.Client.csproj.in

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <LangVersion>7.1</LangVersion>
```

Comment:
Curious: Why specify the language version here?

Reply:
async main